relay.go

```diff
@@ -47,9 +51,9 @@ type Relay struct {
 	relays map[peer.ID]struct{}
 	mx     sync.Mutex
 
-	liveHops map[peer.ID]map[peer.ID]int
-	lhCount  uint64
-	lhLk     sync.Mutex
+	// atomic counters
+	sCount  int32
```

Comment:
Nit: could we give these full names? (`streamCount`?)

Reply:
Sure, will do.

Reply:
done.